src/utils/getBlocksFromTimestamps.ts

```diff
@@ -1,8 +1,16 @@
+// @ts-expect-error ts-migrate(2307) FIXME: Cannot find module '@rainbow-me/apollo/client' or ... Remove this comment to see the full error message
 import { blockClient } from '@rainbow-me/apollo/client';
+// @ts-expect-error ts-migrate(2307) FIXME: Cannot find module '@rainbow-me/apollo/queries' or... Remove this comment to see the full error message
 import { GET_BLOCKS_QUERY } from '@rainbow-me/apollo/queries';
 import logger from 'logger';
 
-async function splitQuery(query, localClient, vars, list, skipCount = 100) {
+async function splitQuery(
+  query: any,
+  localClient: any,
+  vars: any,
+  list: any,
+  skipCount = 100
+) {
```

Comment:
This has all `any` parameters and no return type. From what I can tell it can be something like this, or something similar:

```ts
(
query: (timestamps: number[]) => ApolloQuery
localClient: ApolloClient
vars: any[]
list: number[]
skipCount: number
): Promise<Record<number, { number: number }[]>> {
```